jest.init.js

```diff
@@ -0,0 +1,12 @@
+/*
+ * This file contains the initialization for test cases.
+ * It is being run from jest.config.js as a setupFile.
+ * It gets loaded before the tests are run.
+ */
+
+import { config } from "@vue/test-utils";
+import InlineSvg from "vue-inline-svg";
+
+config.global.components = {
+  InlineSvg,
+};
```

Comment:
I am maybe missing something but I don't see this being used anywhere in the test files - why do we need this? because the files we are testing are using this component and since they won't be referring to main.js that's why?